remote_monitoring_demo/update_results.json

```diff
@@ -70,7 +68,8 @@
          "results": {
            "general_info": {
              "max": 198.50,
-             "mean": 40.1,
+             "avg": 40.1,
```

Comment:
Both `memoryUsage` and `memoryRSS` needs to have `min`, `max`, `avg` and `sum`

Reply:
Done